app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(
+      user: current_user
+    )
+    set_likeable(params[:likeable_type], params[:likeable_id])
+    nil unless @like.save
```

Comment:
I'm not sure what this does for you as you are not rendering any partial. You should be returning a http status 4xx to catch during the AJAX request in case it fails